git/services/gitRepoInfo.js

```diff
@@ -5,7 +5,7 @@
  * @return {Object} An object containing the github owner and repository name
  */
 module.exports = function gitRepoInfo(packageInfo) {
-  var GITURL_REGEX = /^(?:git\+https|https?):\/\/[^/]+\/([^/]+)\/(.+).git$/;
+  var GITURL_REGEX = /^(?:git\+https|https|http?):\/\/[^/]+\/([^/]+)\/([^\n.]+)(?:\.git)?$/;
```

Comment:
This bit of the regex doesn't look right to me:

`/^(?:git\+https|https|http?)...`

This will match

* `git+https`
* `https`
* `http`
and
* `htt`

Reply:
Also this relies upon the url ending with a newline if it doesn't end with `.git`